meca:
  joint_arm1_position_controller:
    type: position_controllers/JointPositionController
    joint: joint_arm1
    

  joint_arm2_position_controller:
    type: position_controllers/JointPositionController
    joint: joint_arm2
    
